src/ansys/sound/core/sound_composer/track.py

```diff
@@ -0,0 +1,337 @@
+                str_source = (
+                    f"{"Data not set" if data is None else data.name}\n\t\tType: SourceAudio"
+                )
+
+            else:
+                match self.source:
+                    case SourceSpectrum():
+                        data = self.source.source_spectrum_data
+                        type_str = "SourceSpectrum"
+
+                    case SourceBroadbandNoise():
+                        data = self.source.source_bbn
+                        type_str = "SourceBroadbandNoise"
+
+                    case SourceBroadbandNoiseTwoParameters():
+                        data = self.source.source_bbn_two_parameters
+                        type_str = "SourceBroadbandNoiseTwoParameters"
+
+                    case SourceHarmonics():
+                        data = self.source.source_harmonics
+                        type_str = "SourceHarmonics"
+
+                    case SourceHarmonicsTwoParameters():
+                        data = self.source.source_harmonics_two_parameters
+                        type_str = "SourceHarmonicsTwoParameters"
+
+                str_source = (
+                    f"{"Data not set" if data is None else data.name}\n"
+                    f"\t\tType: {type_str}\n"
+                    f"\tSource control: "
+                    f"{"Set" if self.source.is_source_control_valid() else "Not set"}"
+                )
+
+        else:
+            str_source = "Not set"
+
+        return (
+            f"{self.name}\n"
+            f"\tSource: {str_source}\n"
+            f"\tFilter: {"Set" if self.filter is not None else "Not set"}"
+        )
+
+    @property
+    def name(self) -> str:
+        """Name of the track."""
+        return self.__name
+
+    @name.setter
+    def name(self, string: str):
+        """Set the track name."""
+        self.__name = string
+
+    @property
+    def gain(self) -> float:
+        """Track gain in dB.
+
+        Gain in dB to apply to the synthesized signal of the track.
+        """
+        return self.__gain
+
+    @gain.setter
+    def gain(self, value: float):
+        """Set the track gain."""
+        self.__gain = value
+
+    @property
+    def source(self) -> AnySourceType:
+        """Source object associated with the track.
+
+        The source of the track is used to synthesize the corresponding signal. Its type can be
+        either :class:`SourceSpectrum`, :class:`SourceBroadbandNoise`,
+        :class:`SourceBroadbandNoiseTwoParameters`, :class:`SourceHarmonics`,
+        :class:`SourceHarmoncisTwoParameters`, or :class:`SourceAudio`.
+        """
+        return self.__source
+
+    @source.setter
+    def source(self, obj: AnySourceType):
+        """Set the track source."""
+        if (obj is not None) and (not (isinstance(obj, AnySourceType))):
+            raise PyAnsysSoundException(
+                "Specified source must have a valid type (SourceSpectrum, SourceBroadbandNoise, "
```

Comment:
@ansaminard why not using the `:class:`SourceBroadbandNoise` syntax like you did in the previous property ?

Reply:
@a-bouth Because this is an error message, not a documentation